aspnet/publishing/iis.rst

```diff
@@ -291,6 +312,17 @@ Troubleshooting
 
 - Confirm that the Application Pool is not in the `Stopped` state.
 
+IISIntegration module was not injected
```

Comment:
Let me confirm this tomorrow by end of day. I'm a little concerned that it doesn't cover the self-contained exception under the same issue. I'll report back when I've checked it out.